backend/src/agents/config/tools.yaml

```
@@ -0,0 +1,55 @@
+  description: >-
+    A tool to refresh or reload the current page in the browser.
+    Use this if the content has changed or if the page needs to be reloaded.
+    no input is required.
+
+
+get_current_url:
+  name: "Get Current page URL"
+  description: >-
+    A tool to fetch the current URL of the active browser page.
+    Use this to find out which page is currently being viewed.
+    No input is required.      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)